data/card_catalogue/upgradeable-1.json

```diff
@@ -1,40 +1,52 @@
 [
   {
-    "title": "Whip",
-    "upgrade": "Pitch",
+    "title": "Toss",
+    "upgrade": "Throw",
     "tier": "blue",
     "details": {
-      "rules": "Throw an object up to 10 squares",
-      "flavour": "If you're going to throw in the towel, at least do it properly.",
-      "image": "assets/core/card/illustration/Whip.png"
+      "rules": {
+        "active": "Throw an object accurately at a target up to 5 metres away.",
+        "passive": ""
+      },
+      "flavour": "Go ahead, toss your hat in the ring.",
```

Comment:
this now matches the illustration!

![Toss](https://user-images.githubusercontent.com/8680290/128256905-397f986f-fc02-4c64-be40-40e0ea6caef3.png)